python/KAPEL.py

```diff
@@ -271,6 +271,9 @@ def record_individual_period(config, results):
         # Only report on pods that have completed. Running pods won't have an endtime
         if not ('starttime' in records and 'endtime' in records):
             continue
+        processors = records.get('cores', 0) or config.processors
+        assert processors > 0, \
+            "must specify pilot CPU count via pod resource requests, or PROCESSORS config var"
```

Comment:
Can we keep it workload-agnostic and just say "CPU count" or "core count" instead of referring to "pilots" ?

Reply:
Yep, updated